src/main/java/marustuff/textometer/model/Metering.java

```diff
@@ -13,43 +12,29 @@
 public class Metering {
     @Id
     private String word;
-
     private int score;
     public Instant createdAt;
-    public void incScore(){
-        score++;
-    }
-
+    private final String emptyWord="none";
     {
         this.createdAt = Instant.now();
     }
 
```

Comment:
czemu tak?

Reply:
https://stackoverflow.com/questions/221611/creation-timestamp-and-last-update-timestamp-with-hibernate-and-mysql